src/main/java/com/crowdin/client/stringtranslations/StringTranslationsApi.java

```diff
@@ -44,20 +44,24 @@ public AlignTranslationResponse alignTranslation(Long projectId, AlignTranslatio
      * @param translationId translation identifier
      * @param limit maximum number of items to retrieve (default 25)
      * @param offset starting offset in the collection (default 0)
+     * @param labelIds filter approvals by labelIds
+     * @param excludeLabelIds exclude approvals by labelIds
      * @return list of approvals
      * @see <ul>
      * <li><a href="https://developer.crowdin.com/api/v2/#operation/api.projects.approvals.getMany" target="_blank"><b>API Documentation</b></a></li>
      * <li><a href="https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.approvals.getMany" target="_blank"><b>Enterprise API Documentation</b></a></li>
      * </ul>
      */
-    public ResponseList<Approval> listTranslationApprovals(Long projectId, Long fileId, Long stringId, String languageId, Long translationId, Integer limit, Integer offset) throws HttpException, HttpBadRequestException {
+    public ResponseList<Approval> listTranslationApprovals(Long projectId, Long fileId, Long stringId, String languageId, Long translationId, Integer limit, Integer offset, String labelIds, String excludeLabelIds) throws HttpException, HttpBadRequestException {
```

Comment:
please keep the `limit` and `offset` parameters at the end of the parameter list for both methods

Reply:
Hi @andrii-bodnar, I didn't realize those should be in the end. I've updated it now :)